### DISCUSSION TITLE
Regions and account types

### DISCUSSION THREAD

**Discussion by ivan-kleshnin** [vercel/vercel#4762] [General] [answered]
Are regions for Serverless-Function for Hobby accounts limited to SFO region?
 
1) This article says YES:
https://vercel.com/knowledge/choosing-deployment-regions
 
2) This page (official docs) has no mention of it:
https://vercel.com/docs/configuration#project/regions

**Comment by mcsdevv** [comment, marked as answer]
Hello @ivan-kleshnin, the article is correct and there's a PR pending to add a note to the section referenced in the documentation: https://github.com/vercel/docs/pull/1924

I think we can make this clearer still though 👍

**Comment by ivan-kleshnin** [comment]
Thanks. I thought **Multiple Regions** mean cases where a customer wants 2+ regions for (different) Serverless Function in a single project.

Does it really also mean a single non-default region? 😮 

Then I think **multiple regions** term is not correct and you should rephrase it to something like **non-default and/or multiple regions** or just **non-default region** because this limitation is a superset of multiple regions.

This limitation (if I understand it correctly) is really harsh. Is there a chance it will be relaxed for `PRO` tier? The latter is really not much different in comparison to `Hobby`, t.b.h.
